src/pages/Bridge/useBridgeModal.ts

```diff
@@ -0,0 +1,63 @@
+import { useCallback } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { AppState } from '../../state'
+import { ecoBridgeUIActions } from '../../services/EcoBridge/store/UI.reducer'
+import { BridgeModalState, BridgeModalStatus } from '../../services/EcoBridge/EcoBridge.types'
+
+//TODO move hook to better place
```

Comment:
This hooks could be inside `/hooks` folder, right?

Reply:
done